checkov/common/bridgecrew/vulnerability_scanning/integrations/docker_image_scanning.py

```diff
@@ -48,6 +48,15 @@ def _create_report(
             }
             for vul in twistcli_scan_result["results"][0].get("vulnerabilities") or []
         ]
+        packages = [
+            {
+                "type": package.get("type"),
+                "name": package.get("name"),
+                "version": package.get("version"),
+                "licenses": package.get("licenses") or [],
+            }
+            for package in twistcli_scan_result["results"][0].get("packages") or []
```

Comment:
can you create a new variable for `twistcli_scan_result["results"][0]` and use it here and above.

Reply:
sure!